projects/common/src/utilities/formatters/enum/display-string-enum.pipe.ts

```diff
@@ -0,0 +1,11 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { displayStringEnum } from './display-string-enum';
+
+@Pipe({
+  name: 'htDisplayStringEnum'
+})
+export class DisplayStringEnumPipe implements PipeTransform {
```

Comment:
Why not use displayString?

Reply:
Hmm. Not sure that will work, or at least it shouldn't. The `displayString` method shouldn't replace `-` or `_`. It should just basically String(<arg>) any input but not format it... with some extra logic for null checks, arrays, etc.

Reply:
I don't think we want to overload display string with this behavior (for example "my-value" would become "My Value", which we don't want the main string pipe doing)